packages/template-typescript-minimal/app/pages/home.tsx

```diff
@@ -90,6 +91,10 @@ const Home = ({value}: Props) => {
         return () => clearInterval(interval)
     }, [counter, setCounter])
 
+    useServerContext(({res: pageRes}) => {
+        pageRes.status(404)
+    })
```

Comment:
I'd like to propose this interface with a callback. Why callback?
- callers don’t need to check whether on server side or not
- it discourages the server context to be used in the content of a component